docs/api.md

```diff
@@ -1141,7 +1141,7 @@ for the authenticated user.
   
   <!--end-response-body-get-accountdevices-pushAuthKey-->
 
-* `pushEndpointExpired`: *boolean, required*
+* `pushEndpointExpired`: *DEVICES_SCHEMA.pushEndpointExpired.optional*
```

Comment:
@philbooth is this unrelated change an expected side-effect of the API doc autogeneration, and if not, any suggestion for how I can stop it from happening?

Reply:
In general, if you want to stop the docs from updating you can either remove the `precommit` script from `package.json` or hack `scripts/write-api-docs.js` to just exit immediately. But also I don't think that's something we should be doing fwiw. Maybe it's a better approach to explicitly run `scripts/write-api-docs.js` in a fresh branch and PR that as a docs-only change, then rebase your own branch after merge (or maybe not, if that seems like too much faffing about).

Anyway, I'm guessing it was maybe disabled before for some reason and that's why your PR is picking up the change?

Reply:
>  Maybe it's a better approach to explicitly run scripts/write-api-docs.js in a
> fresh branch and PR that as a docs-only change, then rebase your own branch after merge

Solid plan, I'm going to do that.  As long as what it's trying to generate here is in fact the desired output.

Reply:
> As long as what it's trying to generate here is in fact the desired output.

Yeah, it's not ideal, I realise. When I get round to fixing #1903, all of the unexpanded indirections in this doc should get magically fixed. In the meantime, it requires a small amount of hoop-jumping for the reader.